### ISSUE TITLE
#7 Fix swipeable component

### ISSUE THREAD

**Issue by dewarian** [sjagoori/bloom#11]
## What is the issue
At the moment the swipe component from [react-swipeable-views](https://github.com/oliviertassinari/react-swipeable-views) has a bug where it takes the entire height of the biggest element.

## Solution
Enable `animateHeight={true} and hope it gets fixed or implement our own hack just like this possible solution: [height not updating properly fix with conditional CSS](https://github.com/oliviertassinari/react-swipeable-views/issues/526#issuecomment-801586106)

### Relevant links

TL;DR It is a known bug that the height doesn't update properly.
[https://github.com/oliviertassinari/react-swipeable-views/issues/526](https://github.com/oliviertassinari/react-swipeable-views/issues/526) 

TL;DR issue where `animateHeight={true}` doesn't properly contain the content
[https://github.com/oliviertassinari/react-swipeable-views/issues/177](https://github.com/oliviertassinari/react-swipeable-views/issues/177)

> _Originally posted by @sjagoori in https://github.com/sjagoori/bloom/pull/7#issuecomment-847736588_

**Comment by dewarian** [comment]
Issue currently moved to the backlog, potentially to sprint 3.
